SkietbaanBE/SkietbaanBE/appsettings.Development.json

```diff
@@ -6,5 +6,8 @@
       "System": "Information",
       "Microsoft": "Information"
     }
+  },
+  "ConnectionStrings": {
+    "SkietbaanDatabase": "Server=dev.retrotest.co.za;Database=skietbaan;User Id=group3;Password=Y%6KG-bmr96Pe5rC;"
```

Comment:
This is sensitive data that you are pushing to a public repository, 
Look at setting up environment variables instead

Reply:
we've been asked to put it here by "francois", but will settup environment variable for future projects.
thanks